vision:
  gate:
    b_max: 156
    b_min: 0
    bilateral_iter: 1
    closing_iter: 1
    closing_mat_point: 1
    g_max: 70
    g_min: 21
    opening_iter: 1
    opening_mat_point: 1
    r_max: 186
    r_min: 0
